### PR TITLE
ROX-21668: Custom namespaces in port-forward scripts

### PR DESCRIPTION
**Pull request by mclasmeier**
## Description

First part of the larger PR https://github.com/stackrox/stackrox/pull/9314.

In an unrelated PR these scripts should be merged as they overlap almost completely.

## Checklist
- [x] Investigated and inspected CI test results
- [ ] Unit test and regression tests added
- [x] ~~Evaluated and added CHANGELOG entry if required~~
- [x] ~~Determined and documented upgrade steps~~
- [x] ~~Documented user facing changes~~

If any of these don't apply, please comment below.

### Here I tell how I validated my change

TODO(replace-me)  
Use this space to explain **how you validated** that **your change functions exactly how you expect it**.
Feel free to attach JSON snippets, curl commands, screenshots, etc. Apply a simple benchmark: would the information you
provided convince any reviewer or any external reader that you did enough to validate your change.

It is acceptable to assume trust and keep this section light, e.g. as a bullet-point list.

It is acceptable to skip testing in cases when CI is sufficient, or it's a markdown or code comment change only.  
It is also acceptable to skip testing for changes that are too taxing to test before merging. In such case you are
responsible for the change after it gets merged which includes reverting, fixing, etc. Make sure you validate the change
ASAP after it gets merged or explain in PR when the validation will be performed.  
Explain here why you skipped testing in case you did so.

Have you created automated tests for your change? Explain here which validation activities you did manually and why so.

### Reminder for reviewers

In addition to reviewing code here, reviewers **must** also review testing and request further testing in case the
performed one does not seem sufficient. As a reviewer, you must not approve the change until you understand the
performed testing and you are satisfied with it.

### CODE REVIEW

File: image/templates/common/port-forward.sh [hunk 1]
```diff
@@ -16,20 +16,22 @@
...
 	exit 1
 fi
 
 KUBE_COMMAND=${KUBE_COMMAND:-{{.K8sConfig.Command}}}
+PORT="$1"
+NAMESPACE=${2:-stackrox}
 
 while true; do
-    central_pod="$(${KUBE_COMMAND} get pod -n 'stackrox' --selector 'app=central' --field-selector 'status.phase=Running' --output 'jsonpath={.items..metadata.name}' 2>/dev/null)"
+    central_pod="$(${KUBE_COMMAND} get pod -n "$NAMESPACE" --selector 'app=central' --field-selector 'status.phase=Running' --output 'jsonpath={.items..metadata.name}' 2>/dev/null)"
```
Comment (tommartensen) [inline image/templates/common/port-forward.sh:29]:
```suggestion
    central_pod="$(${KUBE_COMMAND} get pod -n "${NAMESPACE}" --selector 'app=central' --field-selector 'status.phase=Running' --output 'jsonpath={.items..metadata.name}' 2>/dev/null)"
```

Let's be consistent about the use of "$VAR" -> "${VAR}". Could you update this in the remainder of the PR?

File: image/templates/common/port-forward.sh [hunk 2]
```diff
@@ -16,20 +16,22 @@
 #     $ ./port-forward.sh 8443
 
 if [ -z "$1" ]; then
-	echo "usage: $0 8443"
+	echo "usage: $0 8443 [namespace]"
```
Comment (tommartensen) [inline image/templates/common/port-forward.sh:19]:
```suggestion
	echo "usage: $0 [port] [namespace]"
```

(as far as I understand, port number is configurable as first input parameter to the script)

**Review by tommartensen — CHANGES_REQUESTED** [review]

File: image/templates/common/port-forward.sh (hunk 2 above)
Comment (mclasmeier) [reply image/templates/common/port-forward.sh]: done

File: image/templates/common/port-forward.sh (hunk 1 above)
Comment (mclasmeier) [reply image/templates/common/port-forward.sh]: done

**Review by tommartensen — APPROVED** [review]

**Pull request merged**